guides/v2.10.0/tutorial/simple-component.md

```diff
@@ -184,24 +201,10 @@ Let's call this action `toggleImageSize`
 </article>
 ```
 
-Clicking the anchor element will send the action to the component.
-Ember will then go into the `actions` hash and call the `toggleImageSize` function.
-Let's create the `toggleImageSize` function and toggle the `isWide` property on our component:
-
-```javascript {data-filename=app/components/rental-listing.js data-diff="+5,+6,+7,+8,+9"}
-import Ember from 'ember';
-
-export default Ember.Component.extend({
-  isWide: false,
-  actions: {
-    toggleImageSize() {
-      this.toggleProperty('isWide');
-    }
-  }
-});
-```
+Now when we click on the anchor element, Ember will go into the `actions` hash
+and call the `toggleImageSize` function.
 
-Now when we click the image or the `View Larger` link in our browser, we  see our image show larger.
-When we click the enlarged image we again see it smaller.
+So, when we click the image or the `View Larger` link in our browser, we see our
+image show larger. When we click the enlarged image we again see it smaller.
```

Comment:
This was already missing before, but if you have time, feel free to apply this suggestion in this PR, too:

```suggestion
image shown larger. When we click the enlarged image we again see it smaller.
```